editor/configs/emsRelatedRecordTable.ts

```diff
@@ -12,3 +14,27 @@ export const emsRelatedRecordCols = [
   ALL_EMS_COLUMNS.incident_number,
   ALL_EMS_COLUMNS.crash_match_status,
 ];
+
+/**
+ * Custom mutation variable handler which adds the `_match_event_name` prop
+ * to the record update payload
+ */
+export const getMutationVariables = (
+  _record: EMSPatientCareRecord,
+  column: ColDataCardDef<EMSPatientCareRecord>,
+  value: unknown,
+  defaultVariables: { id: number; updates: Record<string, unknown> }
+): Record<string, unknown> => {
+  if (column.path !== "person_id") {
+    return defaultVariables;
+  }
+  return {
+    id: defaultVariables.id,
+    updates: {
+      ...defaultVariables.updates,
+      _match_event_name: value
+        ? "match_person_by_manual_qa"
+        : "unmatch_person_by_manual_qa",
```

Comment:
gotcha, cool, so this is where the check is for the user-entered value happens. 👍 What UI interactions fire this with no value?